js/client-mainmenu.js

```diff
@@ -394,8 +399,9 @@
 				if (app.ignore[userid]) {
 					$chat.append('<div class="chat">User ' + userid + ' is already on your ignore list. (Moderator messages will not be ignored.)</div>');
 				} else {
-					app.ignore[userid] = 1;
+					app.ignore[userid] = {notified: false};
```

Comment:
```suggestion
					app.ignore[userid] = {notified: true};
```

No need to remind them if they literally just used the command.